Zend/zend_compile.h

```diff
@@ -310,6 +310,13 @@ typedef struct _zend_property_info {
 #define OBJ_PROP_TO_NUM(offset) \
 	((offset - OBJ_PROP_TO_OFFSET(0)) / sizeof(zval))
 
+typedef struct _zend_class_constant {
+	zval value;
+	uint32_t flags;
```

Comment:
maybe you can use zval.u2.var_flags instead of flags here? which should save a pointer size for this struct